### PR TITLE
Earth Tower Prompt

### PR DESCRIPTION
**Pull request by FatePGN**
The prompt says "Choosing a party leader", when the party leader is
actually deciding on whether to stop at the current floor, or move on.

### CODE REVIEW

File: EnglishTranslation-master/ETC.tsv [hunk 1]
```diff
@@ -20780,7 +20780,7 @@ ETC_20160224_020779	The mission for Earth Tower 10F will begin shortly
 ETC_20160224_020780	The first objective has begun{nl}Defeat Gosarius
 ETC_20160224_020781	Gosarius has called for his subordinates
 ETC_20160224_020782	Defeat Gosarius
-ETC_20160224_020783	Choosing a party leader
+ETC_20160224_020783	Party leader is deciding
```
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:20783]: Can you change it to: "The party leader is deciding"

Comment (FatePGN) [inline EnglishTranslation-master/ETC.tsv:20783]: Changed it

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by FatePGN** [comment]
Did another bit, where the refreshment table had incorrect translations for the food buffs, the HP recovery one was actually for SP, vice versa.

**Pull request merged**